js/notebook/src/plot/contextMenu/plotContextMenu.ts

```diff
@@ -30,7 +30,7 @@ export default class PlotContextMenu extends BkoContextMenu {
       ...createSaveAsMenuItems(this.scope),
     ];
 
-    new BeakerXApi(`${window.location.origin}/`)
+    new BeakerXApi(`${(Jupyter.notebook_list || Jupyter.notebook).base_url}`)
```

Comment:
Could you explain why this change fixes the menu issue?

Reply:
There was uncaught exception caused by 404 while requesting /settings endpoint (needed to see if publishing is enabled).

This one is for plots but is exactly the same case.